cmk/gui/plugins/metrics/fortigate_antivirus.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
This belongs into cmk/gui/plugins/metrics/network.py, see https://github.com/tribe29/checkmk/commit/cbb9ea2ba3d58ebc84888888cf6271c3a0dbdf62 (your previous PR)